### DISCUSSION TITLE
Total number of recievers for scanmode

### DISCUSSION THREAD

**Discussion by amane93** [rtl-airband/RTLSDR-Airband#341] [Q&A]
can anyone tell me how many recievers can rtlsdr support for listening in to streams. Currntly mysetup has 3 receivers the stream for the first two Rx can be accessed but gives an error for the 3rd reciever? does it support only 2 or are multiple Rx supported?

**Comment by charlie-foxtrot** [comment]
Not sure I understand the question. Can you provide a config file and the error message you’re seeing?

**Reply by amane93** [reply]
```
devices:
({
 type = "rtlsdr";
 index = 0;
 gain = 30;
 correction = 1;
 mode = "scan";
 channels:
 ( {
  freqs = (120.0,124.3);
  labels = ("120", "124.3" )
   outputs: (
  {
    type = "icecast";
    server = "127.0.0.1";
    port = 8000;
    mountpoint = "stream-0.mp3";   
    name = "Tower"; 
    genre = "ATC"; 
    username = "source";
    password = "hackme";
 send_scan_freq_tags = true; 
 } );
 }
 );
 },
{
 type = "rtlsdr";
 index = 1;
 gain = 30;
 correction = 1;
 mode = "scan";
 channels:
 ( {
  freqs = (125.3);
  labels = ("125.3" )
   outputs: (
  {
    type = "icecast";
    server = "127.0.0.1";
    port = 8000;
    mountpoint = "stream-1.mp3";   
    name = "Tower"; 
    genre = "ATC"; 
    username = "source";
    password = "hackme";
 send_scan_freq_tags = true; 
 } );
 }
 );
 },
{
 type = "rtlsdr";
 index = 2;
 gain = 30;
 correction = 1;
 mode = "scan";
 channels:
 ( {
  freqs = (128.9,130.0);
  labels = ("128.9", "130.0" )
   outputs: (
  {
    type = "icecast";
    server = "127.0.0.1";
    port = 8000;
    mountpoint = "stream-2.mp3";   
    name = "Tower"; 
    genre = "ATC"; 
    username = "source";
    password = "hackme";
 send_scan_freq_tags = true; 
 } );
 }
 ); 
});

```

i have three rtl Rx attached to my system and i want to give different frequencies to each of them to monitor. i can access strem0 and stream1 but i am unable to access stream-2.when I open stream 2 in the browser i get **404 - The file you requested could not be found**

**Reply by amane93** [reply]
and when i open the icecast admin panel there are only two mountpoints generated for stream 0 and stream1

**Reply by charlie-foxtrot** [reply]
That info helps. I think you’re hitting a limit on the icecast side. The default is 2 sources but that can be increased there (see https://icecast.org/docs/icecast-trunk/config_file/)

I would recommend adding a file output to each of those and see if your getting files generated. 

Once you confirm that 3 streams is working you may want to use mixers to avoid scan mode

**Reply by amane93** [reply]
Thank you the default limit of sources was indeed 2 in the icecast.xml file changing that worked thanks!!

**Reply by charlie-foxtrot** [reply]
Excellent!!  Now what are you looking for with your setup?  You have 3 dongles, 5 frequencies, and 3 outputs. Your frequencies are close enough that you could configure your dongles to always be monitoring these 5 frequencies (rather than scanning through them) and combine them all through a mixer into a single stereo stream.
